src/Components/Shared/Header.tsx

```diff
@@ -0,0 +1,10 @@
+import React, { useState } from "react";
+import styles from "./shared.module.css";
+import canvalogo from "../../Media/canvalogo.png";
+
+export const Header: React.FC = () => (
```

Comment:
This filepath could just be `Components/Header/Header.tsx`, since there's only one shared component